src/openms/source/METADATA/ProteinIdentification.cpp

```diff
@@ -81,7 +81,8 @@ namespace OpenMS
     enzyme(UNKNOWN_ENZYME),
     missed_cleavages(0),
     peak_mass_tolerance(0.0),
-    precursor_tolerance(0.0)
+    precursor_tolerance(0.0),
+    digestion_enzyme("unknown_enzyme","")
```

Comment:
why treating the enzyome as a string? wouldnt it make sense to here actually store a pointer or an actual `Enzyme` object? or am I missing something?